src/components/header/export-modal/renderer.tsx

```diff
@@ -102,7 +102,7 @@ class ExportModal extends React.PureComponent<Props, State> {
       content.config = mergeConfig({}, this.props.config, content.config);
     }
 
-    const blob = new Blob([JSON.stringify(content, null, 2)], {
+    const blob = new Blob([JSON.stringify(content, null, 0)], {
```

Comment:
Please use json-stringify-pretty-compact instead.

Reply:
aah nice we want best of both the worlds, i will add the fix

Reply:
also @domoritz do we want that 0 indentation or the previous default 2 indentation

Reply:
We don't need any arguments anymore. Look at other uses of json-stringify-pretty-compact.